integration_tests/src/main/python/array_test.py

```diff
@@ -251,6 +251,23 @@ def test_array_contains_for_nans(data_gen):
             array_contains(col('a'), lit(float('nan')).cast(data_gen.data_type))))
 
 
+# When `data_gen` is `null_gen`, Databricks runtime will throw an exception:
```

Comment:
Not only Databricks, but also Spark after 340, right? Better to update the comment accordingly.